src/services/moderation.service.ts

```diff
@@ -420,22 +427,48 @@ export class ModService {
     }
   }
 
-  // Produces a report summary.
-  // TODO: add warnings and bans metrics.
-  public async getModerationSummary(
-    guild: Guild,
-    username: string
-  ): Promise<MessageEmbed | string> {
-    const id = await Moderation.Helpers.resolveToID(guild, username);
+  // Finds any associated IDs
+  // Returns all Alt IDs and the one given
+  public async getAllKnownAltIDs(guild: Guild, givenID: string) {
+    const altDoc = (await AltTrackerModel.find({})).find((altDoc) =>
```

Comment:
Additionally you can implement a Disjoint Set, stored in the db. That would make this really clean but difficult to undo operations

Reply:
I dont think you can just add a new one each time there is a new alt, because if there are 3 alts, thats 3 Maps with 3 entries.
4 is 4 Maps with 4 entries

O(n^2) space